sidebars.js

```diff
@@ -126,6 +126,7 @@ module.exports = {
       "api/transactions/delete"
     ],
     "Users": [
+      "api/users/overview",
```

Comment:
This page doesnt seem to exist and returns a 404 not found when clicking on the sidebar link

Reply:
I'm not getting a 404 when I click on the sidebar link

Reply:
hmmm ok cool maybe it was just my instance not refreshing the new file. This doesnt seem to be an issue anymore :+1: